dts/espressif/32s2/esp32s2-pinctrl.dtsi

```diff
@@ -3147,7 +3147,6 @@
 
 			uart0_rx_gpio46: uart0_rx_gpio46 {
 				pinmux = <ESP32_PINMUX(46, ESP_U0RXD_IN, ESP_NOSIG)>;
-				bias-pull-up;
```

Comment:
I know GPIO46 is input-only. Where is it written it has no PU support?

Reply:
Here, it is not really about pull-up support on GPIO46 but rather that this pin is fixed to pull-down as per ESP32-S2's online documentation (check first note in [this section](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/api-reference/peripherals/gpio.html#overview)).